exec/java-exec/src/main/java/org/apache/drill/exec/store/sys/zk/ZkAbstractStore.java

```diff
@@ -54,31 +59,26 @@ public ZkAbstractStore(CuratorFramework framework, PStoreConfig<V> config)
     this.prefix = parent + "/";
     this.framework = framework;
     this.config = config;
+    this.childrenCache = new PathChildrenCache(framework, parent, true);
```

Comment:
I consistently use `this` (i) in c'tor assignments or (ii) if there is any shadowing variable.